adage/analyze/tests.py

```diff
@@ -1025,3 +1034,107 @@ def test_expressionvalue_big_post(self):
             self.deserialize(resp)['meta']['total_count'],
             self.expressionvalue_count
         )
+
+
+@override_settings(HAYSTACK_CONNECTIONS=TEST_INDEX)
+class SearchIndexTestCase(ResourceTestCaseMixin, TestCase):
+    searchURI = '/api/v0/search/'
+    experiments = [
+        {
+            'accession':    'E-GEOD-31227',
+            'name':         'Expression data of Pseudomonas aeruginosa isolates'
+                            ' from Cystic Fibrosis patients in Denmark',
+            'description':  'CF patients suffer from chronic and recurrent '
```

Comment:
Hmm, something about the spacing here feels funky. Is there a double space between `'description':` and `'CF patients...'`? Also is the PEP 8 linter cool with the indenting of the lines after this? I feel like it might want them to be aligned with either `'CF....'` or `'description'`. Minor things, but what do you think?